package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "hang-hub",
+  "version": "0.0.1",
```

Comment:
The same question as before: Why `0.0.1`? If we want to set version `0.0.1` we should also update tags with release versions.
Also, it can be problematic whit publishing it on Chrome Web Store and Firefox Add-ons - After every code change, we need to update the version. Currently, there is a `1.0.1` version.
Another question: why you didn't change a version in `backend/package.json`?

Reply:
We don't need to update this version at any time. And the version field is required for package.json file.

Reply:
Why you edited my answer? ;p

Reply:
> And the version field is required for package.json file.

*`manifest.json`. Yes, my bad.

Reply:
@ma2ciek by mistake, sorry. I returned the previous version :P